### ISSUE TITLE
Headline field labels are not translatable

### ISSUE THREAD

**Issue by danielguillan** [getkirby/panel#532]
The following config:

```yaml
some_headline:
  type:  headline
  label:
    en: Headline
    fr: Titre
    es_LA: Título
```

throws the following warnings:

````
Warning: preg_split() expects parameter 2 to be string, array given in …/kirby/toolkit/lib/html.php on line 105

Warning: implode(): Invalid arguments passed in …/kirby/toolkit/lib/html.php on line 105
```

**Comment by RHansmann** [comment]
Already solved: pull request #490. 

The change will be included in the next release. If You need it right away, You must implement the pull request manually by yourself.

**Comment by danielguillan** [comment]
That's awesome. Thanks!
